README.md

```diff
@@ -66,16 +68,16 @@ Keep
 For safe keeping
 ```
 
-Sections are saved when you use `./note-end` to complete your notes for the day.
+Sections are saved when you use `note-end` to complete your notes for the day.
 
 ### Useful tips
-* To quickly open yesterday's note, simply run `./note-yesterday`
-* Same goes for tomorrow's note, simply run `./note-tomorrow`
-* To create a note without updating or syncing your files, use `./note-new` 
-* To open a note without creating a new file and adding sections to it, use `./note-open`
+* To quickly open yesterday's note, simply run `note-yesterday`
+* Same goes for tomorrow's note, simply run `note-tomorrow`
+* To create a note without updating or syncing your files, use `note-new` 
+* To open a note without creating a new file and adding sections to it, use `note-open`
 * All scripts that open notes accept a date as an argument to open the note file of that day.
-    * `./note-open 'today'` (same as `./note-open`)
-    * `./note-new 'last tuesday'`
-    * `./note-tomorrow 'last sunday'` (kinda dumb but it works :shrug:)
-    * `./note-yesterday 'next monday'` (kinda dumb but it works :shrug:)
+    * `note-open 'today'` (same as `./note-open`)
+    * `note-new 'last tuesday'`
+    * `note-tomorrow 'last sunday'` (kinda dumb but it works :shrug:)
```

Comment:
I tried these on my Linux setup and I does not seem to work. It will definitely not work on MacOs and if we want to make the setup fully compatible with this format, might require a bit of work. What do you think, should we leave these here ?

Reply:
`note-tomorrow 'a date'` and `note-yesterday 'a date'` can be removed, thay don't add any value imo.